Adyen/Core/Payment Methods/Abstract/PaymentMethodType.swift

```diff
@@ -180,6 +180,9 @@ public enum PaymentMethodType: RawRepresentable, Hashable, Codable {
         case let .other(value): return value
         }
     }
+}
+
+extension PaymentMethodType {
```

Comment:
What does introduction of extension change? 🤔

Reply:
Body length